### ISSUE TITLE
Log File Spam causing lag in VAB

### ISSUE THREAD

**Issue by MrKiel** [BobPalmer/USI-LS#97]
The more USI-LS parts you have, the worse it gets:

```
ERROR Object reference not set to an instance of an object IN ModuleLifeSupport
 
(Filename: C:/buildslave/unity/build/artifacts/StandalonePlayerGenerated/UnityEngineDebug.cpp Line: 56)

ERROR Object reference not set to an instance of an object IN ModuleLifeSupport
 
(Filename: C:/buildslave/unity/build/artifacts/StandalonePlayerGenerated/UnityEngineDebug.cpp Line: 56)

ERROR Object reference not set to an instance of an object IN ModuleLifeSupport
 
(Filename: C:/buildslave/unity/build/artifacts/StandalonePlayerGenerated/UnityEngineDebug.cpp Line: 56)

ERROR Object reference not set to an instance of an object IN ModuleLifeSupport
```

[output_log USILS.txt](https://github.com/BobPalmer/USI-LS/files/160016/output_log.USILS.txt)

**Comment by BobPalmer** [comment]
Unable to repro - what version of KSP are you using?

**Comment by blueExcess** [comment]
I'm actually getting this error in my log as well. Only seems to happen in the VAB. Generally if I exit out of the VAB and go back in, that will fix it for a while (the lag, don't know if it is logging it still). I haven't noticed a specific part or anything that sets it off - seems to happen whether I've got 3 parts down or 70 parts down, and I haven't noticed any increase in occurrences whether there's more USI parts or less (or none). I can sometimes get it to stop by clicking out of KSP (not running in background), waiting 20 seconds or so, and then re-activating the window. Generally a temporary fix though. 

This happens pretty much every time, except if I make sure KSP is the only app I'm running (literally nothing else except KSP, Activity Manager, and the standard OS X background apps), and I get nearly no lag at all in the VAB (tbh I haven't checked the log though after a smooth run).

Oh, and I'm on KSP 1.0.5, and all the USI mods (and as many of the other mods as possible) are the last version that was compatible with 1.0.5. Hope this helps.

[KSP.log.zip](https://github.com/BobPalmer/USI-LS/files/213722/KSP.log.zip)

**Comment by BobPalmer** [comment]
Yes, this is an issue fixed in 1.1 (closing since this has been fixed)
